### PR TITLE
Yet Another Rework PR

### PR DESCRIPTION
**Pull request by bluegenes**
features:
  - fresh abstract!
  - fresh conclusion!
  - (hopefully) my final round of full-manuscript cleanup before we can share

conclusion and abstract will undoubtedly require additional editing, but I think they are significantly closer to where we want them to be.

### CODE REVIEW

File: content/01.abstract.md [hunk 1]
```diff
@@ -1,9 +1,8 @@
...
-While adoption of these tools can both facilitate and expedite reproducible data analysis, knowledge of and training in these techniques is still lacking.
-Here, we provide a series of tips, tools, and "good enough" practices for leveraging workflow systems to streamline biological analysis.
+As the scale of biological data generation has increased, the bottleneck of research has shifted from data generation to analysis.
+Researchers commonly need to build computational workflows that include multiple analytic tools and require incremental development as experimental insights demand tool and parameter modifications.
+These workflows can produce hundreds to thousands of intermediate files and results that must be integrated for biological insight.
+The maturation of data-centric workflow systems that internally manage resources, software, and conditional execution of analysis steps are reshaping the landscape of biological data analysis, and empowering researchers to conduct reproducible analyses at scale.
```
Comment (taylorreiter) [inline content/01.abstract.md:6]: ya they are!!!!

File: content/04.workflows-and-software.md [hunk 2]
```diff
@@ -12,17 +12,16 @@ When paired with proper software management, fully-contained workflows are scale
...
 **I. Portability** Analyses written in workflow languages (with integrated software management) can be run across computing systems without changes to code.](images/workflow_figure.svg){#fig:workflow}
 
 To properly direct a workflow, workflow systems need to encode information about the relationships between workflow steps.
 In practice, this means that each analysis step must specify the input (or types of inputs) needed for that step, and the output (or types of outputs) being produced.
-This structure results in several added benefits, beyond those mentioned above.
-First, workflows become self-documented; the directed graph produced by workflow systems can be exported and visualized, producing a graphical representation of the relationships between all steps in a pipeline (see **Figure @fig:sgc_workflow**).
+This structure provides several additional benefits.
+First, workflows become minimally self-documented, as the directed graph produced by workflow systems can be exported and visualized, producing a graphical representation of the relationships between all steps in a pipeline (see **Figure @fig:sgc_workflow**).
```
Comment (taylorreiter) [inline content/04.workflows-and-software.md:24]: yes, love "minimally"

File: content/04.workflows-and-software.md [hunk 3]
```diff
@@ -67,7 +66,7 @@ The [Getting started developing workflows](### getting-started-developing-workfl
 
 ### Wrangling Scientific Software
 
-Analysis workflows rely on multiple software packages to generate final results.
+Analysis workflows commonly rely on multiple software packages to generate final results.
```
Comment (taylorreiter) [inline content/04.workflows-and-software.md:69]: accurate :)

File: content/04.workflows-and-software.md [hunk 4]
```diff
@@ -76,26 +75,30 @@ To meet this need, most workflow managers integrate with software management sys
...
-Conda enables lightweight software installation and can be used with the same commands across platforms.
-Alternatively, container solutions like docker and singularity allow for for the entire computational environment to be captured and distributed, including the operating system.
-This ensures that an environment is completely reproducible, and is common for production workflows.
+The Conda package manager has emerged as a leading solution, largely because it handles both cluster permission and version conflict issues with a user-based software environment system, and features a straightforward "recipe" system which simplifies the process of making new software installable (**Figure @fig:conda_figure**).
+Conda enables lightweight software installation and can be used with the same commands across platforms, but can still be impacted by differences in the host operating system.
+Encasing software environments within "containers" that capture and reproduce all other aspects of the runtime environment can enhance reproducibility over time.
```
Comment (taylorreiter) [inline content/04.workflows-and-software.md]: beautiful transition!

File: content/04.workflows-and-software.md [hunk 5]
```diff
@@ -76,26 +75,30 @@ To meet this need, most workflow managers integrate with software management sys
...
-remain.](images/conda_figure.svg){#fig:conda_figure height=8in}
+persist.](images/conda_figure.svg){#fig:conda_figure height=8in}
 
 #### Getting started with software management
 
+**Using software without learning management systems**
+While package managers and containers greatly increase reproducibility, there are a number of ways to test software before needing to worry about installation.
+Some software packages are available as web-based tools and through a series of data upload and parameter specifications, allow the user to interact with a tool that is running on a back-end server.
+Integrated development environments(IDE) like PyCharm and RStudio can manage software installation for language-specific tools, and can be very helpful when writing analysis code.
```
Comment (taylorreiter) [inline content/04.workflows-and-software.md]:
```suggestion
Integrated development environments (IDE) like PyCharm and RStudio can manage software installation for language-specific tools, and can be very helpful when writing analysis code.
```

File: content/05.project-management.md [hunk 6]
```diff
@@ -98,17 +99,17 @@ Interactive visualizations are also helpful in exploratory data analysis.
...
-Some of the tools discussed above will inevitably change over time, but these principles apply broadly and will help you design clear, well-documented, and reproducible analyses. 
-Ultimately, you'll need to experiment with strategies that work for you -- what is most important is to develop a clear set of strategies and implement them tenaciously. 
+In our experience, the best way to have your workflow system work _for_ you is to include as much of your analysis as possible within the automated workflow framework, use self-documenting names, include analysis visualizations, and keep rigorous documentation alongside your workflow that enables you to understand each decision and entirely reproduce any manual steps.
+Some of the tools discussed above will inevitably change over time, but these principles apply broadly and will help you design clear, well-documented, and reproducible analyses.
+Ultimately, you'll need to experiment with strategies that work for you -- what is most important is to develop a clear set of strategies and implement them tenaciously.
```
Comment (taylorreiter) [inline content/05.project-management.md]:
```suggestion
Ultimately, you will need to experiment with strategies that work for you -- what is most important is to develop a clear set of strategies and implement them tenaciously.
```

File: content/05.project-management.md [hunk 7]
```diff
@@ -120,15 +121,17 @@ For example, if working with FASTQ data, you can subsample the first million lin
 
 While there are many more sophisticated ways to subsample reads, this technique should be sufficient for testing each step of a workflow prior to running your full dataset.
 
-**Document your process** 
-Writing documentation in plain text allows your note files to be saved under version control along with your workflow. 
-We recommend using Markdown language for your notes so they can include helpful headings, code formatting, and embedded images. 
-While there are many text editors to choose from, editors with visual formatting previewing such as Hackmd or Atom render previews of text written in Markdown [@url:https://hackmd.io/, @url:https://atom.io/], as do webservices like GitHub.
+**Document your process**
+Document your changes, explorations, and errors as your develop.
```
Comment (taylorreiter) [inline content/05.project-management.md]:
```suggestion
Document your changes, explorations, and errors as you develop.
```

File: content/05.project-management.md [hunk 8]
```diff
@@ -120,15 +121,17 @@ For example, if working with FASTQ data, you can subsample the first million lin
...
+Use  of these notes alongside your workflow.
```
Comment (taylorreiter) [inline content/05.project-management.md]:
```suggestion
```

File: content/06.data-resource-management.md [hunk 9]
```diff
@@ -22,19 +23,20 @@ In others cases, particularly for biologists conducting novel experiments, these
...
 You can find relevant sequencing data either by starting from the "data accessibility" sections of papers relevant to your research or by directly searching for your organism, environment, or treatment of choice in public data portals and repositories.
-The International Nucleotide Sequence Database Collaboration (INSDC), which includes the Sequence Read Archive (SRA), European Nucleotide Archive (ENA), and DataBank of Japan (DDBJ) is the largest repository for raw sequencing data [@doi:10.1093/nar/gkv1323].
+The International Nucleotide Sequence Database Collaboration (INSDC), which includes the Sequence Read Archive (SRA), European Nucleotide Archive (ENA), and DataBank of Japan (DDBJ) is the largest repository for raw sequencing data, but no longer accepts sequencing data from large consortia projects [@doi:10.1093/nar/gkv1323].
+These data are instead hosted in consortia-specific databases, which may require some domain-specific knowledge for identifying relevant datasets and have unique download and authentication protocols.
```
Comment (taylorreiter) [inline content/06.data-resource-management.md:27]: NICE.

File: content/06.data-resource-management.md [hunk 10]
```diff
@@ -62,12 +64,12 @@ Wherever possible, follow the standard guidelines for formatting data for scient
 
 #### Protect valuable data
 
-Other than the workflow itself, raw data are the most important files associated with a workflow as they cannot regenerated if accidentally altered or deleted.
-Keeping a read-only copy of raw data alongside a workflow as well multiple backups protects your data from accidents and computer failure. 
+Aside from the code itself, raw data are the most important files associated with a workflow, as they cannot regenerated if accidentally altered or deleted.
```
Comment (taylorreiter) [inline content/06.data-resource-management.md]:
```suggestion
Aside from the code itself, raw data are the most important files associated with a workflow, as they cannot be regenerated if accidentally altered or deleted.
```

**Review by taylorreiter — APPROVED** [review]

File: content/04.workflows-and-software.md (hunk 4 above)
Comment (bluegenes) [reply content/04.workflows-and-software.md]: 👍!
Forgot to add Gruning 2018 reference here though- could you add after "host operating system"? idk how to edit your edits. doi 10.1016/j.cels.2018.03.014

File: content/05.project-management.md (hunk 8 above)
Comment (bluegenes) [reply content/05.project-management.md:126]: uh oh.. i had some more text changes here

File: content/04.workflows-and-software.md (hunk 4 above)
Comment (taylorreiter) [inline content/04.workflows-and-software.md]:
```suggestion
Encasing software environments within "containers" that capture and reproduce all other aspects of the runtime environment can enhance reproducibility over time [@doi:10.1016/j.cels.2018.03.014].
```

**Pull request merged**